### ISSUE TITLE
Coco detection metrics in object detection tf v.2.2.0

### ISSUE THREAD

**Issue by tazu786** [tensorflow/models#8856]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [X] I checked to make sure that this feature has not been requested already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/blob/master/research/object_detection/colab_tutorials/eager_few_shot_od_training_tf2_colab.ipynb

## 2. Describe the feature you request

I request the implementation of the coco_detection_metrics in the above colab. 

## 3. Additional context

The new colab for few-shot-training you provide is a great improvement on how to easily use the OD api. Having the coco metrics other than only the loss would be great.

## 4. Are you willing to contribute it? (Yes or No)
No

**Comment by pn12** [comment]
Add below in your code to include coco evaluation metrics -

from object_detection.protos import eval_pb2
eval_config = eval_pb2.EvalConfig()
eval_config.metrics_set.extend(['coco_detection_metrics'])

for the precision , recall and iou at different thresholds

I have referred this doc - https://github.com/tensorflow/models/blob/da23acba8ecb8c0e7c9a83cdb9f10092895c9dcc/research/object_detection/g3doc/evaluation_protocols.md
for the same.

**Comment by tazu786** [comment]
thank you @pn12! I kind of have the same issue using model_main_tf2.py with the config file. How can I apply those lines in that case too?
